WebApp/autoreduce_webapp/tests/utils.py

```diff
@@ -74,7 +74,8 @@ def getReductionRun(with_variables=True):
     experiment.save()
     reduction_run = ReductionRun(instrument=instrument, run_number=1, experiment=experiment,
                                  run_version=0, status=StatusUtils().get_queued(),
-                                 script=getValidScript('reduce.py'))
+                                 script=getValidScript('reduce.py'), started_by=-2)
+                                # Not sure what to set started_by too in this test?
```

Comment:
Well done for tracking done this usage. I think though this whole file is actually dead code. Could you look into if the whole `WebApp/autoreduce_webapp/tests/utils.py` file can be deleted. I did a very quick search and I'm not sure it's actually in use. 
If it isn't in use then can you open up a separate Pull request to just delete the whole file? (No need to create a issue). 
If it is being used, then I think `-2` is fine to use

Reply:
Sure thing. I had a look and found that none of the methods were being used outside of the file itself. I presumed this means it is indeed dead code (?)

PR: #477

Reply:
Yep you'll just need to fix up the merge conflict now the other code is removed on dev ( I just merged the PR) 👍